.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: python
+python:
+  - 2.7
+before_install:
+  - export DISPLAY=:99
+  - sh -e /etc/init.d/xvfb start
+install:
+  - pip install -r test-requirements.txt
+  - pip install -r $VIRTUAL_ENV/src/xblock-sdk/requirements/base.txt
+  - pip install -r $VIRTUAL_ENV/src/xblock-sdk/requirements/test.txt
+  - pip install -r $VIRTUAL_ENV/src/xblock/requirements.txt
```

Comment:
This seems fragile, because the xblock and xblock-sdk repos don't expect their requirements to be installed like this.  I would really like to understand why it's necessary to install these, and what the right way to install them is.